specs/spec-1-kudos.md

```diff
@@ -1,106 +1,310 @@
+- React to Kudos 
+
+**Verified Human** logged in user who has credentials issued by `i-am-human`.
 
-Using proof of personhood is not enough to protect a system from abuse. One user could attach a bot to his account and send thousands of kudos to other accounts (potentially to the same human account).
-We propose to limit that activity: Any human can send maximum 14 kudos per week. The quota will reset every Sunday at midday UTC (12:00). The quota will be a parameter and we will be able to change it in the future.
+- Everything User can do
+- Create new Kudos
+- Recieve new Kudos
 
-Moreover, user should be in control of his Kudos. He will have a full right of removing an unwanted Kudo.
+**Administrator** can take additional actions not availalbe to other actors.
 
-Kudo sender will have to set an expire time. To make it more useful, the contract will require that a minimum expire time is 1month.
+- Everything Verified Human can do
+- Delete Kudos
+- Change Settings
+  - Create or remove categories (aka tags)
+  - Adjust experiation time of Kudos
+  - Delete Kudos
 
-### Smart contract management
+## User Stories / Use Cases
 
-Kudo contract will define a set of administrators which will be able to:
+Here are some user stories or use cases for the Kudos widget based on the provided text:
 
-- create or remove categories
-- adjust min expire time
+1. As a **visitor** (not logged in), I want to view the Kudos list, so I can see the various accomplishments and recognitions in the community.
+2. As a **visitor**, I want to view Kudos details, so I can learn more about a specific Kudo and understand its context.
+3. As a **visitor**, I want to share Kudos (list and deep link to a Kudo detail), so I can promote achievements and encourage others to engage with the community.
+4. As a **logged-in user**, I want to comment on Kudos, so I can express my thoughts and engage with the community.
+5. As a **logged-in user**, I want to react to Kudos, so I can show my appreciation and support for the accomplishments.
+6. As a **verified human**, I want to create new Kudos, so I can recognize and celebrate the contributions and achievements of others in the community.
+7. As a **verified human**, I want to receive new Kudos, so I can be acknowledged for my contributions and achievements.
+8. As an **administrator**, I want to delete Kudos, so I can remove inappropriate or unwanted content from the platform.
+9. As an **administrator**, I want to change settings, such as creating or removing categories, adjusting expiration time of Kudos, and deleting Kudos, so I can maintain a positive and productive environment for the community.
 
-### NEAR Social Use Cases
+## Additional Requirements
 
-We define the following features which will use the Social DB smart contract with UI developed in Near Social:
+The following features will use the Social DB smart contract with the Kudos UI developed in Near Social.
 
+### Create a kudo
+
+- Tag any IAH verified account
```

Comment:
could you provide more context? Why kudo should tag other users?

Reply:
I think "tag" is misused here, it should be nominate.